packages/core/.storybook/manager.jsx

```diff
@@ -24,6 +24,13 @@ addons.setConfig({
       }
 
       return <SidebarItem status={parameters.status}>{name.replace(storyStatus, "").trim()}</SidebarItem>;
+    },
+    filters: {
+      patterns: filterInternalStoryDocsPage
     }
   }
 });
+
+function filterInternalStoryDocsPage(item) {
+  return process.env.STORYBOOK_ENV === "development" || !item.tags?.includes?.("internal");
```

Comment:
Can't we filter the entire section without the tags?

Reply:
basically, we can say that if the name includes Internal.
I could add it as an extra check, and leave the current, just for cases where if we would like to have an internal story inside an existing category. wdyt?

Reply:
It should work also with internals stories, no? Anyway I think it's fine for now with tags. It's a new feature in SB7 no? We can use them for the beta/deprecated tags as well

Reply:
- yep it is
- you mean adding styled tags?
- and if I remove the check for `tags` and only exclude the `Internal` category, then it won't exclude internal stories that are inside regular (exported) categories

Reply:
No I mean the sidebar tags, we use the name to decide whether a story is a beta or deprecated, I think using tags is better for that purpose. Let's stay with tags, forget about what I said 😅

Reply:
ohhh I got you
TechDebt task 🙃

Reply:
https://monday.monday.com/boards/3532714909/pulses/6453084060